wagtail/wagtailcore/management/commands/set_url_paths.py

```diff
@@ -1,17 +1,30 @@
 from __future__ import absolute_import, unicode_literals
 
-from django.core.management.base import NoArgsCommand
+
+# Django < 1.10
+try:
+    from django.core.management.base import NoArgsCommand as BaseCommand
+except ImportError:
+    from django.core.management.base import BaseCommand
+
 
 from wagtail.wagtailcore.models import Page
 
 
-class Command(NoArgsCommand):
+class Command(BaseCommand):
+
+    help = 'Resets url_path fields on each page recursively'
+
     def set_subtree(self, root, root_path):
         root.url_path = root_path
         root.save(update_fields=['url_path'])
         for child in root.get_children():
             self.set_subtree(child, root_path + child.slug + '/')
 
-    def handle_noargs(self, **options):
+    def handle(self, *args, **options):
         for node in Page.get_root_nodes():
             self.set_subtree(node, '/')
+
+    def handle_noargs(self, **options):
+        """Django < 1.10"""
+        self.handle(**options)
```

Comment:
Just use the real `BaseCommand` on every Django version and remove this `handle_noargs` method altogether.